packages/inter-protocol/scripts/add-initial-liquidity.js

```diff
@@ -0,0 +1,186 @@
+// @ts-check
+import { AmountMath } from '@agoric/ertp';
+import { makeNotifierFromAsyncIterable } from '@agoric/notifier';
+import { E } from '@endo/far';
+
+const { entries } = Object;
+
+const dec6 = 10n ** 6n;
+const dec4 = 10n ** 4n;
+
+const config = {
+  Central: { IST: 10_000n * dec6 },
+  Secondary: { IbcATOM: 500n * dec4 }, // 20 IST per ATOM
+};
+
+// /** @type {<K extends string,V>(es: [K,V][]) => Record<K,V>} */
+// const recordFromEntries = Object.fromEntries;
+
+const getOfferResult = async (instance, wallet, walletAdmin) => {
+  const offerIt = E(walletAdmin).getOffersNotifier();
+  let result;
+  console.log('awaiting offers...');
+  for await (const offers of makeNotifierFromAsyncIterable(offerIt)) {
+    console.log('offers iterator:', offers);
+    offers
+      .filter(
+        ({ status, invitationQuery }) =>
+          status === 'complete' && invitationQuery.instance === instance,
+      )
+      .forEach(async ({ id }) => {
+        result = await E(wallet).lookup('offerResult', id);
+        console.log({ facet: result });
+      });
+    if (result) break;
+  }
+  return result;
+};
+
+/**
+ *
+ * @param { any } homeP
```

Comment:
not blocking, but do we have a type for this somewhere?

Reply:
It used to be `import('./user-bundle').HomeBundle` which was unresolved, not sure where it came from. And I couldn't find anything else in `agoric-sdk` that defines a proper type for homeP